### DISCUSSION TITLE
more models

### DISCUSSION THREAD

**Discussion by sahilsharma-tmpl** [infiniflow/ragflow#3629] [Q&A]
i wanted to ask if more models of mistral AI (the cheaper ones) will be made available in upcoming times as there are only limited models of mistral ai available and the one we want to use is not present there.

**Comment by devMls** [comment]
Maybe a easy start point is add jina embbeding for other languages:

 https://huggingface.co/jinaai/jina-embeddings-v2-base-es
 https://huggingface.co/jinaai/jina-embeddings-v2-base-fr

or the last one that is multi language:
https://huggingface.co/jinaai/jina-embeddings-v3
....

**Comment by KevinHuSh** [comment]
So, could you list the models that you want?

**Reply by sahilsharma-tmpl** [reply]
ministral 3B and ministral 8b ! can you add these two , they are mistral ai's models only

**Comment by devMls** [comment]
Maybe LLM are heavy for embedding in a rag server, but jina embbedding and reranking are lightweight way to ensure ragflow support multi-lingual

**Comment by KevinHuSh** [comment]
They're added in the latest dev version of docker image.
